app/models/concerns/awardee.rb

```diff
@@ -0,0 +1,23 @@
+module Awardee
+  def acquired_medals
+    @acquired_medals ||= medals_for awardable_content_progress_here.select(&:once_completed?)
+  end
+
+  def unacquired_medals
+    @unacquired_medals ||= medals_for awardable_content_progress_here.reject(&:once_completed?)
+  end
+
+  private
+
+  def medals_for(progress)
+    Organization.current.gamification_enabled? ? progress.map { |i| i.content.medal } : []
```

Comment:
Side note: I think it is always better to not hardcode `Organization.current` in code, but instead place an optional parameter organization with current as default code. That way we do not need to switch to an organization before using the method.

Reply:
Long ago we actually write code like that and then we started using the described approache

Reply:
Sure :+1: I've done that myself, just didn't think of it this time